<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringListEditor</class>
 <widget class="QDialog" name="StringListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="applyButton">
       <property name="toolTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="statusTip">
        <string>Применить и закрыть</string>
       </property>
       <property name="whatsThis">
        <string>Применить и закрыть</string>
       </property>
       <property name="text">
        <string>Применить и закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="toolTip">
        <string>Закрыть</string>
       </property>
       <property name="statusTip">
        <string>Закрыть</string>
       </property>
       <property name="whatsThis">
        <string>Закрыть</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
